.github/workflows/schedule-stale.yml

```diff
@@ -0,0 +1,24 @@
+on:
+  schedule:
+    - cron: "* */12 * * *"
+name: Stale Bot workflow
+jobs:
+  build:
+    name: stale
+    runs-on: ubuntu-latest
+    steps:
+      - name: stale
+        id: stale
+        uses: actions/stale@v3
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+          days-before-stale: 30
+          days-before-close: -1
+          operations-per-run: 300
+          remove-stale-when-updated: true
+          stale-issue-message: |
+            This issue has been automatically marked as stale because it hasn't had any recent activity. It will be closed in 14 days if no further activity occurs.
+            If we missed this issue please reply to keep it active.
+            Thanks for being a part of the Nrwl community! 🙏
+          exempt-issue-labels: |
+            community,community: plugin-submission,type:bug,type:docs
```

Comment:
How come the following are exempt?

* community: plugin-submission
  * This label is used for PRs, not issues and generally hopefully we should get to these.
* type:bug
  * I think bugs can go stale?
* type:docs
  * Not sure why this is exempt.

Should we start with a `only-labels` strategy? We can add a `status: blocked` which would be applied to issues that need more info, need a repro, needs to be retried with latest?

Reply:
I was looking at what gatsby and CRA exempt, and they seem to exempt bugs. But this is our repo so we can do whatever makes sense.

Reply:
Yeah, it depends how the labels are used really.

if `bug` is applied conservatively (when the issue is verified to be a bug) then it makes sense to keep them from going stale until they are fixed.

If `bug` is applied optimistically (say when issues are believed to be a bug by the reporter) then it makes sense to make them stale if they aren't verified to be a bug.

With #3061, I made `type: bug` more of the latter so it automates applying those labels... but it makes it so that `type: bug` might not actually be a bug and should go stale if it's blocked for too long.